package.json

```diff
@@ -26,14 +26,15 @@
     "test:types": "tsd"
   },
   "dependencies": {
-    "@percy/sdk-utils": "^1.0.0",
+    "@percy/sdk-utils": "1.27.0-alpha.0",
```

Comment:
dependency would be `webdriver-utils` here right?

Reply:
`sdk-utils` would anyway be included.

Reply:
Yeah I didn't change it as we are explicitly mentioning it thats why.